todo.sh

```diff
@@ -1142,11 +1142,16 @@ case $action in
 "archive" )
     # defragment blank lines
     sed -i.bak -e '/./!d' "$TODO_FILE"
-    [ "$TODOTXT_VERBOSE" -gt 0 ] && grep "^x " "$TODO_FILE"
-    grep "^x " "$TODO_FILE" >> "$DONE_FILE"
-    sed -i.bak '/^x /d' "$TODO_FILE"
-    if [ "$TODOTXT_VERBOSE" -gt 0 ]; then
-        echo "TODO: $TODO_FILE archived."
+    if grep "^x " "$TODO_FILE"; then
```

Comment:
I think what's lost here is that done tasks are only printed if `TODOTXT_VERBOSE=1`.

Moving the `>> "$DONE_FILE"` up one line and prepending `[ "$TODOTXT_VERBOSE" -gt 0 ] &&` to the following line should fix that.

Reply:
For me, done tasks are printed also with `TODOTXT_VERBOSE=0`: 

```
$ export TODOTXT_VERBOSE=0
$ todo.sh archive 
x Important task @github
```

Reply:
Yes, that's exactly my point. The original code does _not_ print the tasks then.